### ISSUE TITLE
All projects are checked by default

### ISSUE THREAD

**Issue by autarch** [maxmind/App-GHPT#6]
Instead of requiring `--project` CLI arg, we want the default to be to check all projects for started stories.

If this is too slow, it could be done in parallel with `Parallel::ForkManager`.

The `--project` argument should continue to work as-is.

**Comment by 2shortplanks** [comment]
Closing.

Opening new issue #9 for getting Parallel::ForkManager (or some other sort of parallel web request do-dah) to speed things up.
